packages/attendance/src/pages/sms/MessageHistory/Molecule/GetIcon.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
```

Comment:
I think this component has no functional action so it becomes atom not molecule

Reply:
let me check

Reply:
moved it to the atom section